.github/workflows/spotless.yml

```
@@ -0,0 +1,29 @@
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '21'
+          cache: gradle
+
+      - name: Grant execute permission
+        run: chmod +x gradlew
+
+      - name: Run Spotless Check
+        run: ./gradlew spotlessCheck
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)